frameworks/helloworld/tests/test_canary_strategy.py

```diff
@@ -34,6 +34,7 @@ def configure_package(configure_universe):
         sdk_install.uninstall(PACKAGE_NAME)
 
 
+@sdk_utils.dcos_1_9_or_higher
```

Comment:
Are all tests in this module now 1.9 or higher? Is there a way to specify this requirement at a module level instead of per test?